[2021-08-26 01:39:09,886] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test_dag.t1 2021-08-26T01:39:08.562710+00:00 [queued]>
[2021-08-26 01:39:09,897] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: test_dag.t1 2021-08-26T01:39:08.562710+00:00 [queued]>
[2021-08-26 01:39:09,898] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-08-26 01:39:09,899] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-08-26 01:39:09,901] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-08-26 01:39:09,916] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): t1> on 2021-08-26T01:39:08.562710+00:00
[2021-08-26 01:39:09,923] {standard_task_runner.py:52} INFO - Started process 6216 to run task
[2021-08-26 01:39:09,927] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'test_dag', 't1', '2021-08-26T01:39:08.562710+00:00', '--job-id', '24', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpmd7gmy_i', '--error-file', '/tmp/tmp1ir75_tn']
[2021-08-26 01:39:09,929] {standard_task_runner.py:77} INFO - Job 24: Subtask t1
[2021-08-26 01:39:09,979] {logging_mixin.py:104} INFO - Running <TaskInstance: test_dag.t1 2021-08-26T01:39:08.562710+00:00 [running]> on host f913d28eb98c
[2021-08-26 01:39:10,042] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=dini
AIRFLOW_CTX_DAG_ID=test_dag
AIRFLOW_CTX_TASK_ID=t1
AIRFLOW_CTX_EXECUTION_DATE=2021-08-26T01:39:08.562710+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-26T01:39:08.562710+00:00
[2021-08-26 01:39:10,061] {python.py:151} INFO - Done. Returned value was: None
[2021-08-26 01:39:10,072] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=test_dag, task_id=t1, execution_date=20210826T013908, start_date=20210826T013909, end_date=20210826T013910
[2021-08-26 01:39:10,100] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-26 01:39:10,141] {local_task_job.py:151} INFO - Task exited with return code 0
